storybook/src/example-pages/toepassingen/aanvragen/documenten.ts

```diff
@@ -84,18 +91,21 @@ examplePageFactory(
                   <fieldset>
                     <legend class="sr-only">Documenten</legend>
                     <div class="form-group dso-files">
-                      <div class="dso-rich-content">
-                        <p>
-                          Dit is een pagina waar er al documenten zijn toegevoegd zodat je kan zien wat er gebeurt als
-                          er daadwerkelijk formfiles geupload worden. Hoe het er dan uitkomt te zien.
-                        </p>
-                        ${buttonTemplate({
-                          label: "Lees meer",
-                          type: "button",
-                          variant: "tertiary",
-                          icon: { icon: "chevron-down" },
-                        })}
-                      </div>
+                      ${richContentTemplate({
+                        children: html`
+                          <p>
+                            Dit is een pagina waar er al documenten zijn toegevoegd zodat je kan zien wat er gebeurt als
+                            er daadwerkelijk formfiles geupload worden. Hoe het er dan uitkomt te zien.
+                          </p>
+                          ${buttonTemplate({
+                            label: "Lees meer",
+                            type: "button",
+                            variant: "tertiary",
+                            icon: { icon: "chevron-down" },
+                          })}
+                        `,
+                      })}
+
                       <div class="dso-label-container">
```

Comment:
Kan je eens onderzoek doen naar de `formGroupTemplates` zoals `formGroupFilesTemplate()` ? Ik denk dat je hier dan ook nog wel wat kan refactoren

Reply:
Dat kan zeker. Er staat hier nog wel ee nlegend in de fieldset en dat wordt niet aangeboden in het template. Ik weet niet wat daarover gezegd is.

Reply:
Dat weet ik ook niet, goeie vraag. Komen we later wel op terug.